web/modules/custom/dpl_das/src/Elba/Client.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\dpl_das\Elba;
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\RequestOptions;
+use Symfony\Component\Serializer\Encoder\EncoderInterface;
+
+/**
+ * Client for Elba webservices.
+ *
+ * These web services are provided by the Royal Danish Library.
+ *
+ * @see https://webservice.statsbiblioteket.dk/elba-webservices/
+ */
+class Client {
+
+  /**
+   * Constructor.
+   */
+  public function __construct(
+    private ClientInterface $client,
+    private EncoderInterface $encoder,
+    // Note that the uri is all lower case despite the provided documentation.
+    // This is intentional and required for the service to work.
+    private string $uri = "https://webservice.statsbiblioteket.dk/elba-webservices/services/placecopyrequest",
```

Comment:
Good question but I do not think it makes sense to change the current implementation:

1. [The `placeCopyRequest` is the only endpoint the implementation in the ding2 project has called](https://github.com/ding2/ding2/blob/master/modules/ting_das/includes/ting_das.admin.inc#L35) since the module was introduced there >1.5 years ago.
2. It is likely that this entire integration will be moved to FBI as they have also just [implemented Elba support](https://fbi-api.dbc.dk/documentation#Elba-services) and it is probably preferrable to move the implementation there in the future.

If we are yo change this I could split the url into two parts: The base url and `placecopyrequest`.